code/modules/shuttle/supply.dm

```diff
@@ -746,6 +752,85 @@
 
 	frequency.post_signal(src, status_signal)
 
+/obj/machinery/computer/supplycomp/proc/sort_and_send_messages()
```

Comment:
A better name would help here. This name suggests it sorts and sends messages. While it does something else. It sends all players that have an order a message saying their order is denied.
Gonna leave the naming up to you